### DISCUSSION TITLE
Blink Not Working On Raspbian Arduino IDE Version

### DISCUSSION THREAD

**Discussion by digidhamu** [feilipu/Arduino_FreeRTOS_Library#11] [Q&A]
Could you please help me what is wrong in the below code with Raspbian version of Arduino IDE as I could not see build in LED flickering but continuous. But the same code works fine from MacOS version of Arduino IDE.

Please advise.

```
#include <Arduino_FreeRTOS.h>

void TaskBlink( void *pvParameters );

void setup() {

  xTaskCreate(
    TaskBlink
    ,  (const portCHAR *)"Blink"
    ,  128
    ,  NULL
    ,  2
    ,  NULL );
}

void loop() {
}

void TaskBlink(void *pvParameters) {
  (void) pvParameters;

  pinMode(LED_BUILTIN, OUTPUT);

  for (;;)
  {
    digitalWrite(LED_BUILTIN, HIGH);
    vTaskDelay( 1000 / portTICK_PERIOD_MS );
    digitalWrite(LED_BUILTIN, LOW);
    vTaskDelay( 1000 / portTICK_PERIOD_MS );
  }
}
```

**Comment by NicoHood** [comment]
You need to download the latest arduino ide from arduino.cc for arm. The raspbian software is outdated.

**Comment by digidhamu** [comment]
@NicoHood I can see now working after installed the latest version. Thanks for your support.

**Comment by feilipu** [comment]
Good to hear!
